ci/install.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -ex
+
+# Nothing to install.
```

Comment:
Is install automatically run for Elixir projects?

Reply:
Yeah travis always runs a command for it unless you override, and it'll break if there's not a `mix.exs`.

Reply:
([yes](https://docs.travis-ci.com/user/languages/elixir/#default-commands))

Reply:
whoops, didn't see